src/migrators/schema.ts

```diff
@@ -0,0 +1,18 @@
+export const data = {
+    migrations: [
+        {
+            "satellite_body": [
+                "-- The ops log table\nCREATE TABLE IF NOT EXISTS _electric_oplog (\n  rowid INTEGER PRIMARY KEY AUTOINCREMENT,\n  namespace String NOT NULL,\n  tablename String NOT NULL,\n  optype String NOT NULL,\n  primaryKey String NOT NULL,\n  newRow String,\n  oldRow String,\n  timestamp TEXT\n);",
+                "-- Somewhere to keep our metadata\nCREATE TABLE IF NOT EXISTS _electric_meta (\n  key TEXT,\n  value BLOB\n);",
+                "-- Somewhere to track migrations\nCREATE TABLE IF NOT EXISTS _electric_migrations (\n  id INTEGER PRIMARY KEY AUTOINCREMENT,\n  name TEXT NOT NULL UNIQUE,\n  sha256 TEXT NOT NULL,\n  applied_at TEXT NOT NULL\n);",
+                "-- Initialisation of the metadata table\nINSERT INTO _electric_meta (key, value) VALUES ('compensations', 0), ('lastAckdRowId','0'), ('lastSentRowId', '0'), ('lsn', 'MA=='), ('clientId', '');",
```

Comment:
Hmm, why 'MA==' have reappeared? This should be dropped since
0c7410fdbc2ef2e70c2b8377fc9291b70ad9b1e7
as the value have been changed from string to bytes in the protocol.

Reply:
Should the value be 'AAAAAA==' as the patch suggests?

https://github.com/electric-sql/typescript-client/commit/0c7410fdbc2ef2e70c2b8377fc9291b70ad9b1e7#diff-9792734f68fd42ce3c188d374a226f50fa9c3afd9390e9348d1a29db063e1820L37

Reply:
Fixed